apps/worker/tests/test_backfill.py

```
@@ -0,0 +1,415 @@
+    # CodeRabbit finding on #342: without a rollback here, the connection's transaction
+    # stays aborted and the _mark_failed/_requeue_for_retry UPDATE that follows would
+    # itself raise InFailedSqlTransaction, leaving the job stuck in 'processing' forever
+    # instead of being requeued.
+    conn = _FailingConn()
+    events = [_raw_event(id="b-db-error-1")]
+
+    with patch("worker.httpx.Client", return_value=_mock_github_client(events)):
+        worker._handle_backfill_repo_events(conn, 5, _payload(), 0)
+
+    assert conn.rolled_back is True
+    sql, params = conn._cursor.calls[0]
```

Ruff (0.16.1)

[warning] Unpacked variable `params` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)